_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -110,6 +119,42 @@ Heading marked up with `h` element that describes the topic or purpose of its se
 <p>We are open Monday through Friday from 10 to 16</p>
 ```
 
+#### Passed example 7
+
+Heading marked up with `h` element that describes the topic or purpose of a part of its section of the content.
+
+```html
+<h1 class="target">Oranges</h1>
```

Comment:
What is the target class for? It seems redundant.

Reply:
No clue, it comes from the test case I copied+modified. Many of the test cases have `class="target"` on them. I will remove it.